src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Microsoft.Azure.Search.csproj

```diff
@@ -13,7 +13,7 @@
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Spatial" Version="6.15.0" />
+    <PackageReference Include="Microsoft.Spatial" Version="7.2.0" />
```

Comment:
@brjohnstmsft please remove all commented code from .csproj files. They were left for reference purpose during migration. If things are building correctly, they should be removed.
Also remove any traces of project.json, global.json and *.xproj, these files are no longer required.